src/main/groovy/com/github/jengelman/gradle/plugins/shadow/transformers/ManifestAppenderTransformer.groovy

```diff
@@ -60,7 +60,10 @@ class ManifestAppenderTransformer implements Transformer {
     void transform(TransformerContext context) {
         if (manifestContents.length == 0) {
             manifestContents = IOUtil.toByteArray(context.is)
-            IOUtil.close(context.is)
+            try {
+                context.is
+            } catch (IOException ignored) {
```

Comment:
Maybe logging it just in case ?
Should not be a problem with this stream type

Reply:
Will address it in a follow-up.